### PR TITLE
docs: Add docs markdown builds

### PR DESCRIPTION
**Pull request by yasharzolmajdi**
Addresses: https://qctrl.atlassian.net/browse/ACC-441

Changes proposed in this pull request:

- Add docs markdown builds

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```
Comment (leoadec) [inline poetry.lock:1]: Just a small nitpick, but could you update your Poetry to 1.8.3 and rerun this? (There's instructions and rationale for this [here](https://qctrl.slack.com/archives/CF98D2868/p1715641202731539).)

**Review by leoadec — COMMENTED** [review]
Thanks Yash, looks good to me, I just have one nitpick

File: poetry.lock (hunk 1 above)
Comment (yasharzolmajdi) [reply poetry.lock]: Thanks, updated

**Review by leoadec — APPROVED** [review]
Thanks!

**Review by kdav108 — APPROVED** [review]

**Pull request merged**